### ISSUE TITLE
Onion Skin for NCER editor

### ISSUE THREAD

**Issue by Megahog2014** [Garhoogin/NitroPaint#7]
Hello, if it isn't too much trouble to add, would it be possible to add an Onion skinning option for the NCER editor. It would be very useful for dealing with 2D animations for DS games. Animators use the onion skinning to make sure their animations are placed in the right areas, especially for in-between frames. Can't wait to hear from you and have a good one!

**Comment by Garhoogin** [comment]
Hey,

I could certainly implement onion skinning into the NCER editor. It could probably be configured to show a cell translucently in the background, maybe with a configurable opacity. Would it be useful do you think to have multiple frames of it, or just one at a time?
Glad you found this program useful enough to bother with a feature request!

**Comment by Megahog2014** [comment]
Perhaps, make the amount of of frames adjustable. That way people with many computer configurations could figure out what suits best for their needs.

**Comment by Garhoogin** [comment]
Yeah, that might be the best way to go about this. When motivation strikes and I have time I can get to working on this.

**Comment by Megahog2014** [comment]
Hello, I know it's been a while and it might not relate to this issue specifically. But, before you add the feature, do you mind adding in a duplicate frame button. It would make adding frames a bit easier while waiting for the full feature set to come out!

**Comment by Garhoogin** [comment]
I realize that this thread is by now a few years old. The code at the time was messy and I think I knew that I wanted to rewrite it all but did not have sufficient motivation to have done so. Now that the code is in better shape, adding this as a feature is straightforward. Sorry it took so long to get around to, I hope you may still find this useful!
